### DISCUSSION TITLE
Netbox automation with firewall

### DISCUSSION THREAD

**Discussion by jacen2110** [netbox-community/netbox#11925] [Help Wanted!] [answered]
Is it possible to write a script in Netbox to log into my Fortinet firewall, export data and auto populate it into Netbox? If so, are there any out there that I could take a look at?

**Comment by candlerb** [comment, marked as answer]
Anything is possible, but what you've suggested goes against the philosophy of Netbox: see https://docs.netbox.dev/en/stable/introduction/#serve-as-a-source-of-truth

Netbox is not a monitoring system, neither is it intended to be a portal onto data collected by other systems; it's supposed to be a source of truth used to *drive* other systems.
